### PR TITLE
Add CountingWorks PRO

### LINKED ISSUE #2438: Request: CountingWorks PRO
**Name: CountingWorks**
**Website: https://www.countingworks.com/**
**Official resources for icon and color: **
   Need in PNG and SVG 
logo_icon

### PR DESCRIPTION
**Pull request by fbernhart**
countingworkspro

**Issue:** Closes #2438 
**Alexa rank:** [320,529](https://www.alexa.com/siteinfo/countingworkspro.com)

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
- Source: [SVG](https://www.countingworks.com/img/countingworks_logo.svg) from the header of their [blog](https://www.countingworks.com/blog).
- Color: `#2E3084` as used in the wordmark and logo
- **Note:** As the original SVG was pretty a mess, I've manually cleaned it up and removed all the irregularities.

### CODE REVIEW

**Review by PeterShaggyNoble — APPROVED** [review]
> As the original SVG was pretty a mess, I've manually cleaned it up and removed all the irregularities.

This does lead to quite a few inconsistencies with the original but I'm happy to merge it as-is but we'll need approval from at least one of the other @simple-icons/maintainers, too.

**Review by adamrusted — APPROVED** [review]
LGTM! Thanks @fbernhart 🎉

**Pull request merged**
